pkg/infrastructure/azure/azure.go

```diff
@@ -471,7 +476,9 @@ func (p *Provider) InfraReady(ctx context.Context, in clusterapi.InfraReadyInput
 		}
 		logrus.Debugf("created public ip: %s", *publicIP.ID)
 
-		loadBalancer, err := updateExternalLoadBalancer(ctx, publicIP, lbInput)
+		// TODO: spawn thread to wait for machine creation and
+		// associate to new backend pool.
```

Comment:
I think this is done (in a different way) already, right? if so we just need to delete the comment

Reply:
They are eventually joined, yes. Unfortunately, this delays the install time a bit. The new backend pool is added in InfraReady() while the join doesn't happen until PostProvision(). Ideally we want machines added to the pool as they are provisioned.

Reply:
Removed the comment.